tests/coord.test.js

```diff
@@ -51,4 +52,29 @@ test('validate Coord: default values', () => {
     expect(defCoord.validate()).toBe(false);
     expect(defXCoord.validate()).toBe(false);
     expect(defYCoord.validate()).toBe(false);
-})
\ No newline at end of file
+});
+
+test('verify toString output', () => {
+    let c = new Coord();
+
+    for (let i = 0; i < 100; i++) {
+        c.setLocation(i, i);
+        expect(c.toString()).toBe('(' + i + ', ' + i + ')');
+    }
+});
+
+test('comparison of Coord objects', () => {
+    let c1 = new Coord(1, 1);
+    let c2 = new Coord(1, 1);
+    let c3 = new Coord(2, 2);
+
+    expect(c1.is(c2)).toBe(true);
+    expect(c1.is(c3)).toBe(false);
+});
+
+test('comparison of non-Coord objects', () => {
+    let c = new Coord(1, 1);
+    let p = new Port();
+
+    expect(c.is(p)).toBe(false);
```

Comment:
For this test can you give a port the position `(1,1)` and make sure that doesn't match as well since that would be the most likely candidate to accidentally match?

Reply:
I added the position to the Port.